composer.lock

```diff
@@ -1447,12 +1447,12 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/cultuurnet/udb3-php.git",
-                "reference": "ac0db0f0482ce138ab6ebbfa72215453244233b8"
+                "reference": "4ecbc13064bf28c406ae6800766995f9998efec7"
```

Comment:
If this has only been updated _now_, do we actually even check the permissions when creating/updating productions? cc @jdrieghe ?

Reply:
(This commit is the merge of the new permission https://github.com/cultuurnet/udb3-php/commit/4ecbc13064bf28c406ae6800766995f9998efec7)

Reply:
We check the permissions in the command handling if I'm not mistaken, but the tests don't go through that flow.